packages/block-editor/src/components/inserter/style.scss

```diff
@@ -557,26 +565,31 @@ $block-inserter-tabs-height: 44px;
 			box-shadow: 0 0 0 2px $gray-900, 0 15px 25px rgb(0 0 0 / 7%);
 		}
 	}
+}
 
-	.block-editor-inserter__media-panel {
-		height: 100%;
+.block-editor-inserter__media-panel {
+	height: 100%;
+	padding: 0 $grid-unit-20;
 
-		&-title {
-			font-size: calc(1.25 * 13px);
-		}
+	@include break-medium {
+		padding: 0;
+	}
 
-		&-spinner {
-			height: 100%;
-			display: flex;
-			align-items: center;
-			justify-content: center;
-		}
+	&-title {
```

Comment:
Personal nitpick, I'm not a fan of this & pattern in SCSS, I find it harder to parse and search for.

Reply:
I updated in this part, but if we want to update more usages across the project, we should do it in a different PR. Sounds good? Also it seems I had forgotten to remove the `title` styles in the original PR, as we used the tooltip instead.

Reply:
Sounds good.